src/tenants/treesforjane/RegisterTrees/JaneRegisterTrees.tsx

```diff
@@ -278,77 +295,127 @@ export default function RegisterTrees({}: Props) {
-                });
-              }}
-              mapOptions={{
-                customAttribution:
-                  '<a href="https://www.openstreetmap.org/copyright">© OpenStreetMap contributors</a>',
-              }}
-            >
-              {plantLocation ? (
-                <Marker
-                  latitude={plantLocation[1]}
-                  longitude={plantLocation[0]}
-                  offsetLeft={5}
-                  offsetTop={-16}
-                  style={{ left: '28px' }}
-                >
-                  <div className={styles.marker}></div>
-                </Marker>
-              ) : null}
-              <div className={styles.mapNavigation}>
-                <NavigationControl showCompass={false} />
-              </div>
-            </MapGL>
-          )}
+          <MapGL
+            {...mapState}
+            {...viewport}
+            onViewportChange={_onViewportChange}
+            onStateChange={_onStateChange}
+            onClick={(event) => {
+              setplantLocation(event.lngLat);
+              setGeometry({
+                type: 'Point',
+                coordinates: event.lngLat,
+              });
+              setViewPort({
+                ...viewport,
+                latitude: event.lngLat[1],
+                longitude: event.lngLat[0],
+                transitionDuration: 400,
+                transitionInterpolator: new FlyToInterpolator(),
+                transitionEasing: d3.easeCubic,
+              });
+            }}
+            mapOptions={{
+              customAttribution:
+                '<a href="https://www.openstreetmap.org/copyright">© OpenStreetMap contributors</a>',
+            }}
+          >
+            {plantLocation ? (
+              <Marker
+                latitude={plantLocation[1]}
+                longitude={plantLocation[0]}
+                offsetLeft={5}
+                offsetTop={-16}
+                style={{ left: '28px' }}
+              >
+                <div className={styles.marker}></div>
+              </Marker>
+            ) : null}
+            <div className={styles.mapNavigation}>
+              <NavigationControl showCompass={false} />
+            </div>
+          </MapGL>
         </div>
 
         {/* {errorMessage !== '' ? */}
 
         <ContactDetails
           contactDetails={contactDetails}
           setContactDetails={setContactDetails}
+          register={register}
+          errors={errors}
+          setValue={setValue}
         />
 
         <div className={`${styles.formFieldLarge} ${styles.center}`}>
           <p className={styles.formErrors}>{`${errorMessage}`}</p>
         </div>
+
+        <div className={styles.formRow}>
+          <MaterialTextField
+            multiline
+            rowsMax="4"
+            label={t('donate:giftMessage')}
+            variant="outlined"
+            name={'giftMessage'}
+            // onChange={changeGiftDetails}
+          />
```

Comment:
Why do we have this ?

Reply:
![image](https://user-images.githubusercontent.com/65854629/133615057-b5465bde-f77f-4770-addc-38638085cd20.png)

Some messages from Jane will be displayed here. I have not received it yet.